upload-api/html-storacha/index.jsx

```diff
@@ -220,9 +222,24 @@ export const ValidateEmail = ({
         </div>
         {showPricingTable && (
           <div class="box">
-            <p>
-              In order to upload data you need to sign up for a billing plan:
-            </p>
+            {isReferred ? (
+              <>
+                <p>
+                  Congratulations! You are eligible for a free trial of our Lite or Business subscriptions. That means
+                  we won&apos;t charge you anything today.
+                  If you choose a Lite plan, you will get two months for free! If you choose Business, you will get one month for free!
+                  We do need you to provide a valid credit card before we can start your
+                  trial - pick a plan below and complete the checkout flow to get started!
+                </p>
+                <p>
+                  Please note that after your free trial ends, you will be charged 10 USD per month for Lite or 100 USD per month for Business tier.
```

Comment:
**question:** Is it safe to write these values here literally? They won't update if the pricing changes.